<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager_operations</class>
 <widget class="QDialog" name="manager_operations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>分组信息管理</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>迷你简剪纸</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>管理员欢迎你~</string>
     </property>
    </widget>
    <widget class="QPushButton" name="divide_group_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>面试者分组</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="interviewee_divide">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>541</width>
       <height>321</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>分组号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>专业</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>专业班级</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>设置默认参数</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>441</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>面试状态设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>报名开始</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>报名截止</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>面试开始</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>面试结束</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>统计结果结束</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>每场面试人数设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>面试时间设置/mins</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="exec_button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>520</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>执行设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>管理面试官</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>方正剪纸繁体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>面试官管理</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="interviewer_table">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>443</width>
       <height>271</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>面试官编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>面试官姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>登录权限</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>哈希密码</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="search_viewer_button">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>460</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>迷你简剪纸</family>
      </font>
     </property>
     <property name="text">
      <string>权限修改的面试官编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="interviewer_id">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>490</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_button">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>490</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>迷你简剪纸</family>
      </font>
     </property>
     <property name="text">
      <string> 确认修改</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
